cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CruiseControl.java

```diff
@@ -132,6 +136,14 @@
 
     private boolean isDeployed;
 
+    private static final Map<String, String> CUSTOM_POD_LABELS = new HashMap<>();
+    static {
+        String value = System.getenv(CO_ENV_VAR_CUSTOM_CRUISE_CONTROL_POD_LABELS);
+        if (value != null) {
+            CUSTOM_POD_LABELS.putAll(Util.parseMap(value));
```

Comment:
This might be getting a little into semantics, but isn't `CO_ENV_VAR_CUSTOM_CRUISE_CONTROL_POD_LABELS`
technically custom defaults which makes `CUSTOM_POD_LABELS` actually just `DEFAULT_POD_LABELS`
This is sort of backed up by line 251 merging the custom defaults into the user provided labels.
Not a big issue at all, just curious if you agreed/disagreed here.

Reply:
Once you comment, happy to approve.

Reply:
Yes, you are technically right.